packages/styles/scss/patterns/simplelongform/_jumplink.scss

```diff
@@ -0,0 +1,41 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@mixin jumplink {
+    .#{$prefix}--jumplink {
```

Comment:
I'm struggling to understand the need for this file. A lot of duplicate styling.